src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -26,13 +27,14 @@ public static class JsonObject
         private const int maxDepthAllowed = 100;
 
         /// <summary>
-        /// Convert a Json string back to an object
+        /// Convert a Json string back to an object of type PSObject or Hashtable depending on parameter <paramref name="returnHashTable"/>.
         /// </summary>
         /// <param name="input"></param>
+        /// <param name="returnHashTable"></param>
         /// <param name="error"></param>
-        /// <returns></returns>
+        /// <returns>A PSObject or a Hashtable if the <paramref name="returnHashTable"/> parameter is true.</returns>
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")]
-        public static object ConvertFromJson(string input, out ErrorRecord error)
+        public static object ConvertFromJson(string input, bool returnHashTable, out ErrorRecord error)
```

Comment:
We shouldn't change a public API. Please add new overload.

Reply:
Thanks. I did not know PowerShell also exposes some of its classes. I'll re-add the old signature API  that will then internally call the new overload.